.lintstagedrc.mjs

```diff
@@ -1,7 +1,6 @@
 export default {
     '*': 'prettier --write --ignore-unknown',
     'packages/**/*.{ts,tsx,js,jsx}': 'eslint -c packages/.eslintrc.json --fix',
-    'packages/mask/**/*': () => 'gulp locale-kit --sync-keys --remove-unused-keys',
```

Comment:
plz split the remove locale-kit modify to new pr

Reply:
ok